src/tools/pub/pubspec.md

```diff
@@ -398,6 +403,43 @@ the pattern is considered relative to the package's root directory.
   Support for the `false_secrets` field was added in Dart 2.15.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages may showcase their widgets or other visual elements using screenshots.
+
+A package can include up to 10 screenshots. 
+These are specified in the pubspec as a list under the `screenshots` field. 
```

Comment:
Perhaps we can simplify this mentioning, relying on the context of this being the pubspec page?

```suggestion
A package can specify up to 10 screenshots
as a list under the `screenshots` field.
```

Reply:
Done.